dapps/tests/service/package.json

```diff
@@ -4,6 +4,12 @@
     "haml": "0.4.3"
   },
   "devDependencies": {
+    "embarkjs": "^5.0.0-alpha.5",
```

Comment:
`tests/service` isn't a dapp, it's a plugin used by `tests/app` so no need to add `embarkjs[-*]` to `tests/service/package.json`.